tests/addresses_test.js

```diff
@@ -1,7 +1,9 @@
 "use strict";
 
 // standard setup, present in every test
-const test = require("tape");
+const tape = require("tape");
+const _test = require("tape-promise").default;
```

Comment:
If `_test` is never used then this is perfectly valid:

```js
const test = require("tape-promise").default(tape);
```

Admittedly it's not pretty...